templates/case/edit.html.twig

```diff
@@ -21,3 +21,7 @@
     {{ form_end(case_form) }}
 {% endblock %}
 
+{% block javascripts %}
+    {{ parent() }}
+    {{ encore_entry_script_tags('dawa-address-lookup') }}
```

Comment:
This should probably also happen in `templates/case/new.html.twig`.

Reply:
Fixed in https://github.com/itk-dev/naevnssekretariatet/pull/79/commits/640d7047dd77670ff57a1ec8d6f37bbdcc26fce9